### DISCUSSION TITLE
Any plans on compatibility with Chocapic shaders?

### DISCUSSION THREAD

**Discussion by InfernalSpark** [IrisShaders/Iris#142] [Q&A] [answered]
Chocapic shaders are the ones that I frequently use, especially the Toaster Edition ones, which provide an amazing look for surprisingly low frame-drops. I have tried using them and everything becomes black and the sky starts flashing(surprisingly no crashes).
Looking forward to this actually working
Thanks!

**Comment by IMS212** [comment, marked as answer]
Goal is to get sildur’s shaders fully working, and as the project nears that goal more shaders will start to run due to the new features added, there’s not much focusing on one specific shader.
